pkgs/by-name/en/encore/package.nix

```
@@ -0,0 +1,59 @@
+{ fetchFromGitHub
+, buildGoModule
+, callPackage
+, makeWrapper
+, lib
+, ...
+}:
+buildGoModule rec {
+  pname = "encore";
+  version = "1.39.0";
+
+  src = fetchFromGitHub {
+    name = "${pname}-source";
+    owner = "encoredev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-71vzo52vV4VilBnLZxIWDKIY08rfTavMZ57qgi4pip8=";
+  };
+
+  doCheck = true;
+
+  subPackages = [
+    "cli/cmd/encore"
+    "cli/cmd/git-remote-encore"
+    "cli/cmd/tsbundler-encore"
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  CGO_ENABLED = 1;
+
+  postInstall =
+    let
+      goEncore = callPackage ./go.nix { };
+    in
+    ''
+      mkdir -p $out/share/runtimes
+      cp -r $src/runtimes/* $out/share/runtimes
+
+      ln -s ${goEncore}/bin/* $out/bin
+
+      wrapProgram $out/bin/encore \
+        --set ENCORE_RUNTIMES_PATH $out/share/runtimes \
+        --set ENCORE_GOROOT ${goEncore}/share/go \
+        --set GOROOT ${goEncore}/share/go
+    '';
+
+  vendorHash = "sha256-lM03+eBrny7uNKAq4xuQ3HSmX+aglaSEaRCetGgdyjQ=";
+  proxyVendor = true;
+
+  meta = with lib; {
+    description = "Encore is the Backend Development Platform purpose-built to help you create event-driven and distributed systems.";
```

The following suggestions follows recommendations for meta.description as outlined in [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
```suggestion
    description = "Backend Development Platform to create event-driven and distributed systems";
```